python/cuml/test/test_metrics.py

```diff
@@ -71,3 +82,99 @@ def test_sklearn_search():
 
     sk_cu_grid.fit(gdf_data, gdf_train.train)
     assert sk_cu_grid.best_params_ == {'alpha': 0.1}
+
+
+def unit_param(*args, **kwargs):
+    return pytest.param(*args, **kwargs, marks=pytest.mark.unit)
+
+
+def quality_param(*args, **kwargs):
+    return pytest.param(*args, **kwargs, marks=pytest.mark.quality)
+
+
+def stress_param(*args, **kwargs):
+    return pytest.param(*args, **kwargs, marks=pytest.mark.stress)
+
+
+@pytest.mark.parametrize('nrows', [unit_param(30), quality_param(5000),
+                         stress_param(500000)])
+@pytest.mark.parametrize('ncols', [unit_param(10), quality_param(100),
+                         stress_param(200)])
+@pytest.mark.parametrize('n_info', [unit_param(7), quality_param(50),
+                         stress_param(100)])
+@pytest.mark.parametrize('datatype', [np.float32, np.float64])
+def test_accuracy(nrows, ncols, n_info, datatype):
+
+    use_handle = True
+    train_rows = np.int32(nrows*0.8)
+    X, y = make_classification(n_samples=nrows, n_features=ncols,
+                               n_clusters_per_class=1, n_informative=n_info,
+                               random_state=123, n_classes=5)
+    X_test = np.asarray(X[train_rows:, 0:]).astype(datatype)
+    y_test = np.asarray(y[train_rows:, ]).astype(np.int32)
+    X_train = np.asarray(X[0:train_rows, :]).astype(datatype)
+    y_train = np.asarray(y[0:train_rows, ]).astype(np.int32)
+    # Create a handle for the cuml model
+    handle, stream = get_handle(use_handle)
+
+    # Initialize, fit and predict using cuML's
+    # random forest classification model
+    cuml_model = curfc(max_features=1.0,
+                       n_bins=8, split_algo=0, split_criterion=0,
+                       min_rows_per_node=2,
+                       n_estimators=40, handle=handle, max_leaves=-1,
+                       max_depth=-1)
+    cuml_model.fit(X_train, y_train)
+    cu_predict = cuml_model.predict(X_test)
+    cu_acc = cu_acc_score(y_test, cu_predict)
+
+    # sklearn random forest classification model
+    # initialization, fit and predict
+    sk_model = skrfc(n_estimators=40, max_depth=None,
+                     min_samples_split=2, max_features=1.0,
+                     random_state=10)
+    sk_model.fit(X_train, y_train)
+    sk_predict = sk_model.predict(X_test)
+    sk_acc = sk_acc_score(y_test, sk_predict)
+    # compare the accuracy of the two models
+    assert cu_acc >= (sk_acc - 0.07)
```

Comment:
If the `test_accuracy` function is  there to test the `accuracy_score` computation + wrapper itself,  given a set of predicted and ground truth labels, then the comparison should be between `cu_acc_score` and `sk_acc_score` when called for the **same** predicted labels (instead of using `cu_predict` labels for cuml and `sk_predict` labels for skl as the two sets can be different).

If that's not the intention of `test_metrics.py`, just ignore this comment!